types/is-buffer/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for is-buffer 2.0
+// Project: https://github.com/feross/is-buffer#readme
+// Definitions by: Junyoung Choi <https://github.com/rokt33r>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types='node' />
+
+declare function isBuffer(obj: any): boolean;
```

Comment:
This should be:
```typescript
declare function isBuffer(obj: any): obj is Buffer;
```

Reply:
I fixed it and added a test for it. Thanks for the reviewing. 😄